### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by dabeshouse**
Some commits from the big VR branch that stand on their own. Together.

### CODE REVIEW

File: core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java [hunk 1]
```diff
@@ -95,59 +92,63 @@ protected Color4f getDefaultMarkerColor() {
...
+    initialize();
+    sgVisuals = new Visual[] {createLensVisual(),
+        createCylinderVisual("Camera Cylinder 1", RADIUS),
+        createCylinderVisual("Camera Cylinder 2", RADIUS * 3),
+        createBoxVisual()};
 
-    this.sgAppearance = new SimpleAppearance();
-    this.sgAppearances = new SimpleAppearance[] {sgAppearance};
+    addLaserSightLine();
+    updateViewGeometry();
```
Comment (jennej) [inline core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:100]: assuming you meant to drop setViewingAngle here?

**Comment by jennej** [comment]
thank you for your service, removing all those unused empty classes.

**Review by jennej — APPROVED** [review]

File: core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java (hunk 1 above)
Comment (dabeshouse) [reply core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:100]: Yes. Those values are now set in initialize() and don't need to be set again.

Comment (dabeshouse) [reply core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:100]: Hmmm… now it is never used. Guess I could delete it.

Comment (dabeshouse) [reply core/story-api/src/main/java/org/lgna/story/implementation/PerspectiveCameraMarkerImp.java:100]: Deleted it and a few related lines.

**Pull request merged**